server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -566,66 +529,15 @@ export class AssetService {
     return { filepath: asset.resizePath, contentType: 'image/jpeg' };
   }
 
-  private async streamFile(filepath: string, res: Res, headers: Record<string, string>, contentType?: string | null) {
-    await fs.access(filepath, constants.R_OK);
-    const { size, mtimeNs } = await fs.stat(filepath, { bigint: true });
-
+  async streamFile(res: Res, filepath: string, contentType?: string | null) {
     if (contentType) {
-      res.header('Content-Type', contentType);
+      res.contentType(contentType);
     }
-
-    const range = this.setResRange(res, headers, Number(size));
-
-    // etag
-    const etag = `W/"${size}-${mtimeNs}"`;
-    res.setHeader('ETag', etag);
-    if (etag === headers['if-none-match']) {
-      res.status(304);
-      return;
-    }
-
-    const stream = createReadStream(filepath, range);
-    return await pipeline(stream, res).catch((err) => {
-      if (err.code !== 'ERR_STREAM_PREMATURE_CLOSE') {
+    const options = filepath.startsWith('/') ? {} : { root: '/usr/src/app' };
+    res.sendFile(filepath, options, (err) => {
```

Comment:
It looks like we removing support for range support. I don't think you will be able to seek in a video without loading the entire file in the client.

Reply:
Range is handled in `sendFile`: [relevant section](https://github.com/pillarjs/send/blob/b69cbb3dc4c09c37917d08a4c13fcd1bac97ade5/index.js#L645C5-L645C5).

Reply:
Oh sweet! I was looking for this code, hoping that's how it work, but couldn't actually find that section. Looks great!